sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt

```diff
@@ -534,6 +534,14 @@ internal class NativeMapImpl(val map: MapInterface) :
     return map.setStyleGeoJSONSourceData(sourceId, data)
   }
 
+  override fun setStyleGeoJSONSourceData(
```

Comment:
Hmm the geojson with data-id be present in 10.11?

Reply:
Done.